README.md

```diff
@@ -277,25 +336,29 @@ The SIOPv2 Authorization Request is encoded as a URI before being returned to Mo
 
 #### Grants 
 | Field                                                  | Description                                                                                             | Required | References                                                                                                               | Comments |
 | :----------------------------------------------------- | :------------------------------------------------------------------------------------------------------ | :------- | :----------------------------------------------------------------------------------------------------------------------- | :------- |
-| `urn:ietf:params:oauth:grant-type:pre-authorized_code` | Grant Type that allows the Mobile Wallet to follow a Pre-Authorized Code Flow to collect the credential | y        | [OID4VCI](https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#section-4.1.1-4.2.1) |          |
+| `urn:ietf:params:oauth:grant-type:pre-authorized_code` | Grant Type that allows the Mobile App to follow a Pre-Authorized Code Flow to collect the credential | y        | [OID4VCI](https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#section-4.1.1-4.2.1) |          |
 
 
 #### Grant Type: urn:ietf:params:oauth:grant-type:pre-authorized_code
 | Field                 | Description                                                                                              | Required | References                                                                                                                 | Comments |
 | :-------------------- | :------------------------------------------------------------------------------------------------------- | :------- | :------------------------------------------------------------------------------------------------------------------------- | :------- |
-| `pre-authorized_code` | The code representing the Credential Issuer's authorization for the Mobile Wallet to obtain a credential | y        | [OID4VCI](https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#section-4.1.1-4.2.2.1) |          |
+| `pre-authorized_code` | The code representing the Credential Issuer's authorization for the Mobile App to obtain a credential | y        | [OID4VCI](https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#section-4.1.1-4.2.2.1) |          |
 
 > [!WARNING] 
 > TODO: explain rationale behind providing `credential_offer` at this stage
 
-## IDV
+## IDV Collection
+
+The IDV Request sent to the Mobile App contains a `url` field. The Mobile App MUST load this URL in an embedded 
```

Comment:
> The Mobile App MUST

This made me realize, we have `url` marked as required in the IDV Request table (above) but if the SIOPv2 Auth Response includes a compliant `vp_token` then the user won't go through IDV Collection, yeah?

Reply:
Fair question. Don't think the `vp_token` has been thought about deeply yet.

I think it would be possible to make `url` optional in a future spec version, but probably not possible to go the other way?